package.json

```diff
@@ -19,6 +19,9 @@
   },
   "devDependencies": {
     "@types/node": "^9.4.7",
+    "dotenv": "^6.0.0",
```

Comment:
doesn't seem like this gets used

Reply:
It does. but, you have to have a .env file at your root and these are never checked in.  This will set proper environment variables and is easier to maintain than updating .bashrc on windows/mac and each of your environments if you have multiple like I do.